README.md

```diff
@@ -56,10 +55,10 @@ Prometheus will now be reachable at http://localhost:9090/.
 
 ### Building from source
 
-To build Prometheus from the source code yourself you need to have a working
+To build Prometheus from source code, first ensure that have a working
 Go environment with [version 1.13 or greater installed](https://golang.org/doc/install).
-You will also need to have [Node.js](https://nodejs.org/) and [Yarn](https://yarnpkg.com/)
-installed in order to build the frontend assets.
+You also need [Node.js](https://nodejs.org/) and [Yarn](https://yarnpkg.com/)
+installed in order to build the front-end assets.
```

Comment:
frontend is the usual spelling

Reply:
Depends where you are I guess.  When researching my book I came to a different conclusion.  Minor, backing out.